.github/workflows/docker-image.yml

```diff
@@ -8,10 +8,25 @@ on:
 jobs:
 
   build:
+    strategy:
+      matrix:
+        arch: [ amd64, arm64 ]
+      fail-fast: false
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Build the Docker image
-      run: docker build . --file Dockerfile --tag whist-server:$(date +%s)
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          file: ./Dockerfile
+          platforms: linux/${{ matrix.arch }}
+          tags: |
+            whist-team/whist-server:${{ matrix.arch }}-test
```

Comment:
wouldnt it make more sense to use the commit hash here?

Reply:
Yes, but the image is obsolete after the run